src/Commands/Monitor/HealthModels/Entity/EntityGetHealthCommand.cs

```diff
@@ -0,0 +1,101 @@
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using System.Text.Json.Nodes;
+using AzureMcp.Arguments.Monitor.HealthModels.Entity;
+using AzureMcp.Models.Argument;
+using AzureMcp.Models.Command;
+using AzureMcp.Services.Interfaces;
+using ModelContextProtocol.Server;
+
+namespace AzureMcp.Commands.Monitor.HealthModels.Entity;
+
+public sealed class EntityGetHealthCommand : BaseMonitorHealthModelsCommand<EntityGetHealthArguments>
+{
+    public override string Name => "gethealth";
+
+    public override string Description =>
+         $"""
+        Gets the health of an entity from a specified Azure Monitor Health Model.
+        Returns entity health information.
+        
+        Required arguments:
+        - {ArgumentDefinitions.Monitor.Health.Entity.Name}: The entity to get health for
+        - {ArgumentDefinitions.Monitor.Health.HealthModel.Name}: The health model name
+        """;
+
+    public override string Title => "Get the health of an entity in a health model";
+
+    protected override void RegisterOptions(Command command)
+    {
+        base.RegisterOptions(command);
+        command.AddOption(_entityOption);
+        command.AddOption(_healthModelOption);
+        command.AddOption(_resourceGroupOption);
+    }
+
+    protected override void RegisterArguments()
+    {
+        base.RegisterArguments();
+        AddArgument(CreateEntityArgument());
+        AddArgument(CreateHealthModelArgument());
+        AddArgument(CreateResourceGroupArgument());
+    }
+
+    protected override EntityGetHealthArguments BindArguments(ParseResult parseResult)
+    {
+        var args = base.BindArguments(parseResult);
+        args.Entity = parseResult.GetValueForOption(_entityOption);
+        args.HealthModelName = parseResult.GetValueForOption(_healthModelOption);
+        args.ResourceGroup = parseResult.GetValueForOption(_resourceGroupOption);
+        return args;
+    }
+
+    [McpServerTool(Destructive = false, ReadOnly = true)]
+    public override async Task<CommandResponse> ExecuteAsync(CommandContext context, ParseResult parseResult)
+    {
+        var args = BindArguments(parseResult);
+
+        try
+        {
+            if (!await ProcessArguments(context, args))
+            {
+                return context.Response;
+            }
+
+            var service = context.GetService<IMonitorHealthModelService>();
+            var result = await service.GetEntityHealth(
+                args.Entity!,
+                args.HealthModelName!,
+                args.ResourceGroup!,
+                args.Subscription!,
+                args.AuthMethod,
+                args.Tenant,
+                args.RetryPolicy);
+
+            context.Response.Results = ResponseResult.Create<JsonNode>(result, JsonSourceGenerationContext.Default.JsonNode);
+        }
+        catch (Exception ex)
+        {
+            HandleException(context.Response, ex);
```

Comment:
We should log a error here, along the lines of:
 _logger.LogError(ex, "Error getting health entity {Entity Identifier/Details}", args.Entity);
            
 feel free to update it to fit the context here.